subjects/findpairs/README.md

```diff
@@ -0,0 +1,32 @@
+## Find Pairs with Sum
+
+### Instructions
+
+Write a program that finds all pairs of elements in an integer array that sum up to a given target value. The program should output a list of pairs, each representing the indices of the elements that form the pair.
+
+In this exercise you must take in consideration the following:
+
+- Ensure its possible to have positive or negative integers in the array
+- Ensure each element is used only once in a pair
+- Allow for multiple pairs to sum up to the target value
+- Return the message "No pairs found." when no pair is present
+- Return the message "Invalid target sum." if the target is invalid
+- Return the message "Invalid number." if the number in the array is invalid
```

Comment:
What about a wrong format that is not included in the above? For example:
```console
go run . "1, 2, 3, 4" "2"
```
or 
```console
go run . "[1, 2, 3, 4" "5"
```
Is this a valid input or not?

Reply:
added some new info in the subject and test for this cases